### PR TITLE
tt 1.0.0 (new formula)

### PR DESCRIPTION
**Pull request by better0fdead**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

Opened new PR, since i messed old one. Sorry for that.

### CODE REVIEW

**Review by chenrui333 — APPROVED** [review]
LGTM

File: Formula/tt.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  desc "Command-line utility to manage Tarantool applications"
+  homepage "https://github.com/tarantool/tt"
+  url "https://github.com/tarantool/tt/releases/download/v1.0.0/tt-1.0.0-complete.tar.gz"
+  sha256 "6f42d30b9d9f9fbad1907a49cf3394f71c2c86e3faa1194fd9372e4d8877c792"
+  license "BSD-2-Clause"
+
+  depends_on "go" => :build
+  depends_on "mage" => :build
+  depends_on "pkg-config" => :build
+  depends_on "openssl@3"
```
Comment (SMillerDev) [inline Formula/tt.rb:11]: Aren't these needed? https://github.com/tarantool/tt#dependencies

Comment (better0fdead) [reply Formula/tt.rb]: Forgot about command-specific dependencies. Thx, added.

File: Formula/tt.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  depends_on "wget"
```
Comment (SMillerDev) [inline Formula/tt.rb:12]: Since they seem interchangeable this is better
```suggestion
  uses_from_macos "curl"
```

Comment (better0fdead) [reply Formula/tt.rb]: Thx, changed

**Review by SMillerDev — APPROVED** [review]

**Pull request closed without merging**